FamilySearchIosSampleApp/LoginVC.swift

```diff
@@ -193,11 +199,15 @@ class LoginVC: UIViewController {
         if (segue.identifier == "segueToTabBar")
         {
             let tabBarController : UITabBarController = (segue.destinationViewController as? UITabBarController)!
-            
+            tabBarController.tabBar.items![0].title = NSLocalizedString("tabAncestorsName", comment: "name for list tab")
+            tabBarController.tabBar.items![1].title = NSLocalizedString("tabMemoriesName", comment: "name for memories tab")
+
             let treeTVC : TreeTVC = (tabBarController.viewControllers![0] as? TreeTVC)!
```

Comment:
```
guard let treeTVC = tabBarController.viewControllers.first as? TreeTVC else {
    fatalError("The first viewController in the tabBarController should be an instance of TreeTVC")
}
```
You can use the approach above as well where you grab a reference to the tabBarController.  This will still crash the app if you get the wrong type of view controller back but it will crash with a much more useful error message.